### ISSUE TITLE
Error trying to build a Swift pod

### ISSUE THREAD

**Issue by dimitrijevic** [kikettas/xamapod#3]
FIRST AWSOME TOOL!!!

I tried to build a TunnelKit pod and got this error:

    ❌  There was an error binding the library.
     Are there types to be bound in the SwiftyBeaver-Swift.h file?

I understand Swift isn't fully supported but I'm trying to figure out if there is anything I can do in the process to use this pod as I don't see the _SwiftyBeaver-Swift.h_ file (is the error just the template). TIA

**Comment by kikettas** [comment]
When trying to build that pod in Xcode I get this error: 
`linking against a dylib which is not safe for use in application extensions`

Does this happen to you?

**Comment by dimitrijevic** [comment]
Yes, it did, but it manages, after setting _export SWIFT_VERSION=3_, to build dll from source...it might not be the same...I've built it successfully from source [here](https://github.com/keeshux/TunnelKit) (but it still uses SwiftyBeaver in that latest source though). I've attached it as proof, but unfortunately, sharpie didn't generate much of extern method calls...

[TunnelKit.zip](https://github.com/kikettas/Xamapod/files/2869469/TunnelKit.zip)
